.env-devel

```diff
@@ -35,10 +35,10 @@ RABBIT_PORT=5672
 
 
 REGISTRY_AUTH=True
-REGISTRY_PW=z43
+REGISTRY_PW=adminadmin
 REGISTRY_SSL=True
 REGISTRY_URL=masu.speag.com
-REGISTRY_USER=z43
+REGISTRY_USER=admin
```

Comment:
very good!

Reply:
add user/pass for dev should default to these